src/main/plugins/org.dita.base/build_preprocess_template.xml

```diff
@@ -69,13 +69,21 @@ See the accompanying LICENSE file for applicable license.
     </delete>
   </target>
   
-  <target name="ditaval-merge" unless="skip.ditaval.preprocess"
-    dita:depends="{depend.preprocess.ditaval-merge.pre}"
+  <target name="ditaval-merge.init">
+    <condition property="skip.ditaval.preprocess" value="true">
```

Comment:
Thinking more on it, the only thing it does is say "If there is no list of ditavals, don't bother with this step". Using the `skip.` property like with other steps is rather misleading.